swayfmt/src/utils/language/attribute.rs

```diff
@@ -12,21 +13,42 @@ use sway_ast::{
 };
 use sway_types::{constants::DOC_COMMENT_ATTRIBUTE_NAME, Spanned};
 
-impl<T: Format> Format for Annotated<T> {
+impl<ItemKind: Format + Spanned> Format for Annotated<ItemKind> {
```

Comment:
Just to clarify, this declares a generic type `ItemKind` bound by `Format + Spanned` and then implements `Format` for this `Annotated` generic type. Was this the intent here?

If you just want to implement for `Annotated<ItemKind>` (i.e. the `ItemKind` type from sway-ast), then that would be:
```suggestion
impl Format for Annotated<ItemKind> {
```
but if you still want to keep this generic and just add the `Spanned` requirement, this might be clearer as:
```suggestion
impl<T: Format + Spanned> Format for Annotated<T> {
```